backend/src/main/java/gov/cdc/usds/simplereport/config/GraphQlSchemaDirectiveConfig.java

```diff
@@ -1,17 +1,33 @@
 package gov.cdc.usds.simplereport.config;
 
+import gov.cdc.usds.simplereport.api.DefaultArgumentValidation;
 import gov.cdc.usds.simplereport.api.directives.RequiredPermissionsDirectiveWiring;
 import graphql.kickstart.tools.boot.SchemaDirective;
+import graphql.validation.rules.OnValidationErrorStrategy;
+import graphql.validation.rules.ValidationRules;
+import graphql.validation.schemawiring.ValidationSchemaWiring;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
 @Configuration
 public class GraphQlSchemaDirectiveConfig {
   public static final String REQUIRED_PERMISSIONS_DIRECTIVE_NAME = "requiredPermissions";
+  public static final int MAXIMUM_SIZE = 256;
```

Comment:
Personally, I think this limit is fine since the `symptoms` field is specifically increased below.